src/ggml.c

```diff
@@ -11801,7 +11696,9 @@ static bool ggml_compute_forward_mul_mat_use_blas(
 }
 #endif
 
-static void ggml_compute_forward_mul_mat(
+// legacy multiplication matrix just float 32 data type
+
+static void ggml_compute_forward_mul_mat_f32_f32(
```

Comment:
I don't get why this was changed?
This function does all types of matrix multiplications (F32, F16, Q) - it's not just F32 x F32

Reply:
The current conv1d and conv2d need fp16 x fp16 = fp32 for reduce memory usage and precision.